packages/terra-markdown/package.json

```diff
@@ -21,17 +20,13 @@
     "url": "https://github.com/cerner/terra-core/issues"
   },
   "homepage": "https://github.com/cerner/terra-core#readme",
-  "devDependencies": {
-    "terra-toolkit": "^0.x"
-  },
   "peerDependencies": {
     "react": "^15.4.2",
-    "react-dom": "^15.4.2",
-    "terra-mixins": "^1.0.0"
+    "react-dom": "^15.4.2"
   },
   "dependencies": {
-    "classnames": "^2.2.5",
-    "terra-mixins": "^1.0.0"
+    "marked": "^0.3.6",
+    "github-markdown-css": "^2.4.1"
```

Comment:
I moved these 2 dependencies from the root package.json into the terra-markdown package.json.  Both of these dependencies are imported in the `src` files so they need to be in this package.json. 

https://github.com/cerner/terra-core/blob/master/packages/terra-markdown/src/Markdown.jsx#L3-L4